### ISSUE TITLE
Traduccion de URLs

### ISSUE THREAD

**Issue by xavier506** [lacchain/eosio.lacchain.net#67]
Las urls no corresponden al idioma

**Comment by xavier506** [comment]
![image](https://user-images.githubusercontent.com/5632966/122300093-149dee00-cebc-11eb-8160-56323fcddfff.png)

**Comment by AngeloCG97** [comment]
Voy a investigar si docusaurus ya lo maneja, por que en primer instancia lo que usan para identificar cada documento es su id y así saber cual es el equivalente en otro idioma, y el id es lo que usan como ruta en la documentación.

**Comment by AngeloCG97** [comment]
https://github.com/facebook/docusaurus/discussions/5006

**Comment by xavier506** [comment]
awesome jib reaching out to them on GH , I would disagree that localizing slugs has little SEO value, but at least we k now its not available yet

**Comment by AngeloCG97** [comment]
Correcto, dejemos el issue por acá para darle seguimiento en otras versiones de docusaurus.
